### ISSUE TITLE
Isaac Sim semantic segmentation masks appear pixelated

### ISSUE THREAD

**Issue by sersandre** [isaac-sim/IsaacSim#682]
### Description

When using Isaac-sims replicator to generate semantic segmention masks of objects, their segmentation masks appear strangely pixelated: 

<img width="960" height="641" alt="Image" src="https://github.com/user-attachments/assets/40d49b0e-e6ac-4b2c-85ed-dfd109c4a1c5" />

I ran replicator code basically ported over from earlier versions of isaac sim (both 5.1.0 and 5.0.0), where this worked fine:
```
def run_replicator(self):
        rep.orchestrator.set_capture_on_play(False)
        rep.functional.create.scope(name="Cameras", parent="/World")
        cam = rep.functional.create.camera(
            name="MainCam",
            parent="/World",
            focus_distance=400,
            focal_length=24,
            clipping_range=(0.1, 10000000)
        )

        cam_rp = rep.create.render_product(cam, RESOLUTION, name="main")
        backend = rep.backends.get("DiskBackend")
        backend.initialize(output_dir=os.path.join(os.getcwd(), "outputs"))
        writer = rep.writers.get("BasicWriter")
        writer.initialize(
            backend=backend,
            rgb=True,
            semantic_segmentation=True,
            colorize_semantic_segmentation=True,
            #distance_to_image_plane=True,
            #camera_params=True
        )
        writer.attach(cam_rp)
        timeline = omni.timeline.get_timeline_interface()
        timeline.play()
        for _ in range(200):
            simulation_app.update()
        for i in range(NUM_CAPTURES):
            print("Caputuring frame {} / {}".format(i+1, NUM_CAPTURES))
            cam_rp.hydra_texture.set_updates_enabled(True)
            randomize_camera(cam, self.offset)
            rep.orchestrator.step(rt_subframes=RT_SUBFRAMES)
            cam_rp.hydra_texture.set_updates_enabled(False)

        rep.orchestrator.wait_until_complete()
        writer.detach()
        cam_rp.destroy()
```

Spawning the objects in the scene is done via:
```
 for key in self.object_dict:
            prim_path = "/World/" + key
            prim = self.stage.DefinePrim(prim_path, "Xform")
            asset_path = self.object_dict[key]["path"]
            prim.GetReferences().AddReference(asset_path)

            translation = self.object_dict[key]["translation"]
            x, y, z, w = np.array(self.object_dict[key]["orientation"], dtype=float)
            orientation = np.array([w, x, y, z])
            geom_prim = GeomPrim(str(prim.GetPath()),
                                 apply_collision_apis=True,
                                 translations=translation + self.offset,
                                 orientations=orientation,
                                 scales=[0.001, 0.001, 0.001],
                                 reset_xform_op_properties=True)
            geom_prim.set_collision_approximations(["convexHull"])
            rigid_prim = RigidPrim(str(prim.GetPath()))
            rigid_prim.set_masses([1.0])
            self.rigid_objects.append(rigid_prim)
            add_labels(prim, labels=key, taxonomy="class")
```
  Any reason as to why this behaviour occurs?

### Isaac Sim version

6.0.1

### Operating System (OS)

Ubuntu 22.04

### GPU Name

RTX 3080

### GPU Driver and CUDA versions

Driver 595.71.05 Cuda 13.2

**Comment by hawkeyeking** [comment]
Hi @sersandre — The sparse, "see-through" scatter you see is the DLSS denoiser artifact. Isaac Sim ships DLSS in Performance mode by default, which may produce incorrect transparency on thin geometry. Your objects are scaled to 0.001. IsaacSim 5.0/5.1 didn't surface this the same way, which is why the ported code was clean there.

Try switching DLSS to Quality mode once at startup, before your Replicator setup:
```
pythonimport carb
carb.settings.get_settings().set("rtx/post/dlss/execMode", 2)  # DLSS Quality

```

This is the documented SDG fix ([6.0.0 SDG Workflows tutorial — DLSS Quality Mode](https://docs.isaacsim.omniverse.nvidia.com/6.0.0/replicator_tutorials/tutorial_replicator_sdg_workflows.html#dlss-quality-mode)). Two things to check alongside it:

Resolution — the default Performance mode is worst below ~600×600. What is your RESOLUTION? Bumping it up (or using Quality mode above) should resolve it.

rt_subframes — since you randomize the camera each capture, use rt_subframes=8 so the renderer converges before the annotator reads the mask.

If masks are still degraded after DLSS Quality + rt_subframes=8, please attach the kit_*.log and your RESOLUTION/RT_SUBFRAMES values and I'll dig in as a possible 6.0.1 regression.

As an alternative to DLSS entirely, you can switch anti-aliasing to FXAA: carb.settings.get_settings().set("/rtx/post/aa/op", 2).

**Comment by hawkeyeking** [comment]
Hello!

As there haven’t been any recent responses, we’re closing this issue or discussion to help keep our forum organized.

If you’re still encountering this issue or have further questions, please feel free to open a new issue or discussion with updated details. When doing so, referencing or linking to this original discussion would be helpful for context.

Thank you for contributing to the NVIDIA Isaac Sim community!

Best regards,
The NVIDIA Isaac Sim Team
